test/integration/test_b2_command_line.py

```diff
@@ -133,8 +133,7 @@ def read_from(self, f):
 
 def remove_warnings(text):
     return os.linesep.join(
-        line for line in text.split(os.linesep)
-        if 'DeprecationWarning' not in line and 'UserWarning' not in line
+        line for line in text.split(os.linesep) if 'DeprecationWarning' not in line
```

Comment:
why this change?

Reply:
Missing zoneinfo caused `UserWarning`. We don't need this anymore